docs/beginner-checks.md

```diff
@@ -0,0 +1,77 @@
+
+To change the default setting for token permissions, add the following to the top of your workflow:
+
+```
+permissions:
+contents: read
+```
+
+When you add a GitHub Action, be sure to read the Action’s docs to see if it needs any additional permissions; this information is usually prominent in the Action’s README.
+
+## 2. Manage contributions to your project
+
+As projects grow, they generally start including contributions from others. Contributors can expand your project’s scope and maturity, but they can also introduce security risk. To protect your project at this stage, we recommend improving the Branch Protection check, which allows you, the maintainer, to define rules that require certain workflows for certain branches.
+
+### Branch Protection reduces the risks of errors and hacks
+
+The [Branch Protection check](https://github.com/ossf/scorecard/blob/main/docs/checks.md#branch-protection) can help protect your code from unvetted changes. You can choose any or all of the following options:
+
+- Require code review. Select this if your project has more than one maintainer. Requiring review before changes are merged is one of the strongest protections you can give your code. This will also improve your [Code Review check](https://github.com/ossf/scorecard/blob/main/docs/checks.md#code-review) score.
+- Require [status checks](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/collaborating-on-repositories-with-code-quality-features/about-status-checks). All projects would benefit from selecting this option. It ensures that all Continuous Integration (CI) tests have passed before a change is accepted, helping you catch mistakes early on in the development process. This will also improve your [CI Test check](https://github.com/ossf/scorecard/blob/main/docs/checks.md#ci-tests) score.
+- [Require signed commits](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/managing-rulesets/available-rules-for-rulesets#require-signed-commits). All projects would benefit from selecting this option. By requiring that you sign your own commits, you help leave an auditable trail showing that you were the one who made the changes, not a malicious actor who gained access to your project. Requiring signed commits can be especially helpful for single maintainers who don’t have fellow contributors to review their code.
```

Comment:
Signed commits aren't actually judged by scorecard, should we be recommending them here?

Reply:
I definitely see what you mean. I've removed the bullet point about signed commits.